app/processing/account-codes/get-line-code.js

```diff
@@ -0,0 +1,7 @@
+const getLineCode = (description) => {
```

Comment:
I'm not sure if this name is descriptive enough - I was confusing it with get-codes-for-line

Reply:
Renamed to make it clearer it's getting a code from the description